modules/apps/blogs/blogs-service/src/main/java/com/liferay/blogs/service/impl/BlogsEntryLocalServiceImpl.java

```diff
@@ -2293,6 +2298,16 @@ private void _pingTrackbacks(
 		}
 	}
 
+	private String _sanitizeUrlTitle(String urlTitle) {
+		if (urlTitle.startsWith(StringPool.SLASH)) {
+			urlTitle = urlTitle.substring(1);
+
+			return _sanitizeUrlTitle(urlTitle);
+		}
+
+		return urlTitle;
+	}
```

Comment:
@robertoDiaz Can you remove the recursion here? This can be implemented with a simple loop.